ht.access

```diff
@@ -36,15 +36,15 @@ RewriteRule "/\.|^\.(?!well-known/)" - [F]
 # Rewrite secure requests properly to prevent SSL cert warnings, e.g. prevent 
 # https://www.domain.com when your cert only allows https://secure.domain.com
```

Comment:
To me it would make more sense if we also use the `example.com` domain here.

Reply:
Done.